clients/python/tests/test_client.py

```diff
@@ -71,6 +71,22 @@ async def test_register_new_using_s3_uri_builder(client: ModelRegistry):
     assert ma.uri == uri
 
 
+@pytest.mark.e2e
```

Comment:
I suggest we leave these tests as is for now. I just want to remark for future work - It could be nice to test client logic like this with a mock server or something similar, so it doesn't have to be e2e. Seems like we have a pretty loose definition of "end-to-end" in general. There are caveats my suggestion too, and this isn't really a huge problem right now, but I wanted to mention it.